### ISSUE TITLE
package dependencies not installing automatically

### ISSUE THREAD

**Issue by jshamoun** [adokter/bioRad#12]
while installing bioRad in R I recieved the following error:
```
> install_github("adokter/bioRad")
Downloading GitHub repo adokter/bioRad@master
from URL https://api.github.com/repos/adokter/bioRad/zipball/master
Installing bioRad
"C:/PROGRA~1/R/R-33~1.0/bin/x64/R" --no-site-file --no-environ --no-save --no-restore --quiet CMD INSTALL  \
  "C:/Users/jshamou1/AppData/Local/Temp/RtmpsjV7bM/devtools17344a633b4/adokter-bioRad-df64641" --library="C:/Program Files/R/R-3.3.0/library" --install-tests 

* installing *source* package 'bioRad' ...
** R
** data
*** moving datasets to lazyload DB
** inst
** preparing package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called 'spam'
ERROR: lazy loading failed for package 'bioRad'
* removing 'C:/Program Files/R/R-3.3.0/library/bioRad'
Error: Command failed (1)
```
I tried rerunning this after installing Docker but still recieved the same error

**Comment by jshamoun** [comment]
After a lot of trial and error I had to install a range of packages manually. Thus every time there was a statement such as "there is no package called 'xxx'" I installed in seperately and repeated the installation process.

**Comment by adokter** [comment]
Thanks Judy, it seems the package dependencies are not installed automatically, I will look into it

**Comment by adokter** [comment]
I cannot reproduce this error on Mac OSX using a clean R without any packages installed.

**Comment by adokter** [comment]
I can reproduce this error on Windows - unclear why this happens. I added install instructions to install the packages causing this problem manually before installing bioRad, which fixes this issue. But a more elegant solution would be nice.

**Comment by Rafnuss** [comment]
FWI, I still had the same problem today on RStudio Windows.

**Comment by adokter** [comment]
Thanks @Rafnuss, could you provide a copy of the error messsage you got?

**Comment by Rafnuss** [comment]
Unfortunately, I closed the console. But it was exactly like the first message, error after 
```
** preparing package for lazy loading
```

With the same message
```
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called '...'
```

So I got to install manually all theses packages: spam, maps, RgoogleMaps, geosphere, gstable, scales, lazyeval, tibble,jpeg and a few more. I've started on a new Rstudio install with almost no packages installed before. 

Hope this helps,!
Cheers,
